Neos.Fusion/Classes/Core/FusionSourceCodeCollection.php

```diff
@@ -52,12 +52,6 @@ public static function tryFromFilePath(string $filePath): self
         return self::fromFilePath($filePath);
     }
 
-    public static function tryFromPackageRootFusion(string $packageKey): self
-    {
-        $fusionPathAndFilename = sprintf('resource://%s/Private/Fusion/Root.fusion', $packageKey);
-        return self::tryFromFilePath($fusionPathAndFilename);
-    }
```

Comment:
this method was removed as its considered internal and relatively new code from myself :D
The `FusionAutoIncludeHandler` will handle that now.